--- __FILE__
+++ __FILE__
@@ -1,10 +1,21 @@
-<?php declare(strict_types=1);
+<?php
+
+declare(strict_types=1);
+
 namespace Instapro;
-class AReallyBadFile {
-protected string $string = '';
-public function __construct(Foo $a, Bar $bar){}
 
-public function getMethod():string{
-return $this->string;
+class AReallyBadFile
+{
+    protected string $string = '';
+
+    public function __construct(
+        Foo $a,
+        Bar $bar,
+    ) {
+    }
+
+    public function getMethod(): string
+    {
+        return $this->string;
+    }
 }
-}
\ No newline at end of file
